homeassistant/components/sensor/filter.py

```diff
@@ -44,6 +48,8 @@
 DEFAULT_PRECISION = 2
 DEFAULT_FILTER_RADIUS = 2.0
 DEFAULT_FILTER_TIME_CONSTANT = 10
+DEFAULT_LOWER_BOUND = -math.inf
+DEFAULT_UPPER_BOUND = math.inf
```

Comment:
I would avoid using math.inf (saving the math import) by just defaulting to None

Reply:
I would like to keep this (or switch to float('inf') - saving a math import - or sth similar) because this is mathematically more elegant (the lower bound *is* negative infinite and thus not existant). With None more ugly comparisons are included in the source code.

Reply:
```python
if lower_bound and value < lower_bound:
    return value
```

Reply:
lets keep python 3.5 math.inf

Reply:
I disagree. If there is no bound, we shouldn't check it.

```python
if lower_bound is not None and value < lower_bound
    return value
```

Reply:
Does None pass the validity check for float values in voluptuous?

Reply:
Yes, Just mark the key as `vol.Optional` and not give any default.